convert2rhel/checks.py

```diff
@@ -712,7 +711,13 @@ def is_loaded_kernel_latest():
         logger.warning("Skipping the check as no internet connection has been detected.")
         return
 
-    cmd = ["repoquery", "--setopt=exclude=", "--quiet", "--qf", "%{BUILDTIME}\\t%{VERSION}-%{RELEASE}\\t%{REPOID}"]
+    cmd = [
+        "repoquery",
+        "--setopt=exclude=",
+        "--quiet",
+        "--qf",
+        "%{BUILDTIME}\\t%{NAME}-%{VERSION}-%{RELEASE}\\t%{REPOID}",
+    ]
```

Comment:
Note for future @pr-watson: This will (possibly) conflict if https://github.com/oamg/convert2rhel/pull/687 get merged first.